main/libxml2/template.py

```diff
@@ -33,6 +39,10 @@ def post_extract(self):
 
 
 def post_install(self):
+    # A side effect of --enable-static is the production of
```

Comment:
no need for a long comment like that, just say something like `delete spurious files introduced by --enable-static` or something

Reply:
Made comment a single line.